velox/exec/GroupingSet.cpp

```diff
@@ -174,26 +174,15 @@ void GroupingSet::addInputForActiveRows(
   if (ignoreNullKeys_) {
     // A null in any of the keys disables the row.
     deselectRowsWithNulls(*input, keyChannels_, activeRows_, execCtx_);
-    for (int32_t i = 0; i < hashers.size(); ++i) {
-      auto key = input->loadedChildAt(hashers[i]->channel());
-      if (mode != BaseHashTable::HashMode::kHash) {
-        if (!hashers[i]->computeValueIds(*key, activeRows_, lookup_->hashes)) {
-          rehash = true;
-        }
-      } else {
-        hashers[i]->hash(*key, activeRows_, i > 0, lookup_->hashes);
-      }
-    }
-  } else {
-    for (int32_t i = 0; i < hashers.size(); ++i) {
-      auto key = input->loadedChildAt(hashers[i]->channel());
-      if (mode != BaseHashTable::HashMode::kHash) {
-        if (!hashers[i]->computeValueIds(*key, activeRows_, lookup_->hashes)) {
-          rehash = true;
-        }
-      } else {
-        hashers[i]->hash(*key, activeRows_, i > 0, lookup_->hashes);
+  }
+  for (int32_t i = 0; i < hashers.size(); ++i) {
```

Comment:
Why is this change? I didn't expect any changes to the hash aggregation operator itself, only to the min/max_by aggregate functions.

Reply:
This looks like just code cleanup

Reply:
> This looks like just code cleanup

@xiaoxmeng Is this the case? If so, would you submit it in a separate PR?

Reply:
Yeah, it is just a code cleanup. Will put it in a separate PR.